common/bookmarks/00_bookmarks.txt

```diff
@@ -28,19 +28,83 @@ bm_the_invasion_from_draenor = {
 	character = {
```

Comment:
583
- Missing CoA's
![image](https://user-images.githubusercontent.com/74790101/223278896-622c4bcb-e749-4334-b75b-996d77a12c70.png)

603/605
- Ner'zhuls image is cut off (not a biggie, but if we can we should fix)
- Missing CoA's
![image](https://user-images.githubusercontent.com/74790101/223278664-d4f638b0-57f5-4f34-8025-58e8d51adaaf.png)
![image](https://user-images.githubusercontent.com/74790101/223278783-c5b67e67-9deb-4978-8d95-139fb1921a06.png)